about.html

```diff
@@ -0,0 +1,94 @@
+            <li><a href="#">Home</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+    </header>
+    <div class="top">
+        <div class="top1">
+            <img alt="logo" src="Images/Logo.png">
+            <ul class="nav">
+                <li class="nav-item">
+                <a class="nav-link" href="index.html">Home</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="about.html">about</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Sponsors</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link active" href="#">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="main">
+        <div class="overlay">
+            <div class="menu-overlay" onclick = "showMenu()">
+                <div class="menu">
+                    <div class="login">Login</div>
+                    <a class="menu-link" href="index.html">Home</a>
+                    <a class="menu-link" href="about.html">about</a>
+                    <a class="menu-link" href="#">Join</a>
+                    <a class="menu-link" href="#">Sponsors</a>
+                    <a class="menu-link" href="#">Contact</a>
+                </div>
+            </div>
+            <div class="navigation"  onclick = "showMenu()">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            
+
+            <section class="intro" id="Intro">
+                <h1 class="main-title">THE WORLD  FOOTBALL GOVERNING BODY (FIFA)<br> HAS SETTLED ON RWANDA TO HOST THE 73rd FIFA CONGRESS</h1>
+                <div class="intro-message">
+                    <p>The Fédération internationale de football association is the international governing body of association football, beach soccer, and futsal. It was founded in 1904 to oversee international competition among the national associations of Belgium, Denmark, France, Germany, the Netherlands, Spain, Sweden and Switzerland</p>
+                </div>
+            </section>
+        </div>
+    </div>
+    
+    <section id="third">
+        <div class="guide_bar"></div>
+        <div class="intro-message">
+            <p>Take a look for this vidio</p>
+        </div>
+        <div class="meeting">
+            <div class="meet">
+                <iframe width="330" height="250" src="https://www.youtube.com/watch?v=4N-VAJKa7hI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] This section contains a centered column that includes a section header, a divider, a line of text, and two images with text centered over them, please kindly fix this section.
- [ ] In mobile this section images are organized in a vertical column and in desktop images are organized in a horizontal row 🙃 🙃 

> |     | Your design                     | Original design                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="854" alt="Screen Shot 2023-02-16 at 00 12 39" src="https://user-images.githubusercontent.com/74501523/219165481-d1a520db-dc24-4f66-8ea9-cd485e0c2f33.png"> | <img width="873" alt="Screen Shot 2023-02-16 at 00 12 09" src="https://user-images.githubusercontent.com/74501523/219165672-3af8a4ee-ead0-4bfb-a0d0-abe1ea1f49b6.png"> |